app/lib/features/news/widgets/news_side_bar.dart

```diff
@@ -47,13 +49,40 @@ class NewsSideBar extends ConsumerWidget {
     );
 
     return Column(
+      mainAxisAlignment: MainAxisAlignment.end,
       children: <Widget>[
-        const Spacer(),
-        LikeButton(
-          likeCount: news.likesCount().toString(),
-          style: style,
-          color: fgColor,
-          index: index,
+        SizedBox(
+          height: 75,
+          width: 75,
+          child: LikeButton(
+            isLiked: isLikedByMe.valueOrNull ?? false,
+            likeCount: reactions.valueOrNull != null
```

Comment:
`likesCount` this should probably not be a `String` but an `int`, or ... it is a `count` afterall...

Reply:
this should probably be changed to:
```suggestion
            likeCount: reactions.valueOrNull?.length ?? 0
```

Reply:
Fixed by https://github.com/acterglobal/a3/pull/1208/commits/4286329716aadd590bbbb1306d9331d4e3934b6c